tests/unit/StaffNotation.spec.ts

```diff
@@ -86,7 +162,6 @@ describe('StaffNotation', () => {
 
   it('renders drop-down menu correctly - full list, all allowed actions', async () => {
     const wrapper = mount(StaffNotation, { vuetify })
-
```

Comment:
Please stop removing whitespace in code. It's usually there to visually separate one code block from another. Also on line 182 below.